app-web/chinadns-ng/autobuild/defines

```diff
@@ -2,3 +2,5 @@ PKGNAME=chinadns-ng
 PKGSEC=net
 PKGDES="The next generation of chinadns, refactoring with epoll and ipset"
 PKGDEP="glibc ipset"
+
+ABSPLITDBG=0
```

Comment:
This should be marked noarch.

Reply:
It should not be noarch since it requires compilation.